test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -237,6 +240,7 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
can we remove this?

Reply:
I'm not sure, the file was already in the project

Reply:
I´m doing some test.

Reply:
yes, I already remove this line